misk/src/main/kotlin/misk/web/WebActionModule.kt

```diff
@@ -67,14 +72,17 @@ internal class BoundActionProvider<A : WebAction, R>(
     val httpMethod: HttpMethod
 ) : Provider<BoundAction<A, *>> {
 
-    @Inject lateinit var interceptorFactories: List<Interceptor.Factory>
+    @Inject lateinit var userProvidedInterceptorFactories: List<Interceptor.Factory>
+    @Inject lateinit var defaultInterceptorFactories: @MiskDefault List<Interceptor.Factory>
```

Comment:
I don’t think I like this name

Reply:
`miskInterceptorFactories`? I think we'll end up with a few layers of Interceptors like OkHTTP, so we'll probably have to play around with it then.